### DISCUSSION TITLE
Extra field default value does not work?

### DISCUSSION THREAD

**Discussion by a-mian-lateral** [chamilo/chamilo-lms#5459] [General]
Hello, I have a problem with extra fields for the user, I have to set a default value for some of them but, even if I set a value, the form input is not pre-compiled. So I thought that the value would have been saved during the process of creating the new user but is not.
My question is: is this feature working? If so, what should I put inside the default value to make it working?

Thanks in advance

**Comment by Coursenligne** [comment]
Yes it works.
First of all could you please state the Chamilo version you are running?
Then giving a bit more of inputs would help, such as what do you want to achiev, what kind of field did you add, the value you tried....
I created extra fileds for learning coins, and I can set a default value (number of coins at registration) and it works well.

**Reply by a-mian-lateral** [reply]
Hello, thanks for the reply. Sorry I forgot the version, I'm running the 1.11.26.
The field is the one to force a user to change password at the next login.

I set the id of the field to `ask_new_password`, possible values are 0 or 1, and default value is 1.
When I manually set to 1 it works but the "yes" option is not pre-selected, and if I dont select anything at all it doesn't set the value at 1 so the application does not ask a new password at the next login of that user.
Basically I'm forced to select the "yes" value when I create a user, otherwise it doesn't ask a new password.

**Comment by Coursenligne** [comment]
I may be wrong, but I remembre changing a setting in configuration file prior to selecting the extra field.
If you search for password, you'll find the variable (someting like Force_Change_Password_FirstLogin... This is set to 0, you may change it to 1 and then this will be requested at the very first login.
